style.css

```diff
@@ -327,14 +327,14 @@ ul li ul {
     padding: 0 0 0 1.5em;
 }
 dt {
-    width: 140px;
```

Comment:
This was a change to avoid breaking the definition list I added to the Docs (twig variable list), so we might want to check that this doesn't cause it to look too broken.